docs/src/main/asciidoc/spring-cloud-contract.adoc

```diff
@@ -10,21 +10,22 @@
 = Spring Cloud Contract
 
 _Documentation Authors: Adam Dudczak, Mathias Düsterhöft, Marcin Grzejszczak, Dennis Kieselhorst, Jakub Kubryński, Karol Lassak,
-Olga Maciaszek-Sharma, Mariusz Smykuła, Dave Syer_
+Olga Maciaszek-Sharma, Mariusz Smykuła, Dave Syer_, Jay Bryant
```

Comment:
:+1: :+1: :+1: :+1: :+1: :+1: 

I tried to keep the names in alphabetical order. WDYT?

Reply:
I think only a few people look at the list of authors. That said, I think at least some people think the primary author goes first, followed by whomever made the next biggest contribution, and so on. Looked at that way, you should be first, and I should be last. If I alphabetize, people may think I'm the primary author, which would be a false impression. I just swept the floors. You built the house.

I'll hold off on putting my name first, until you've had a chance to consider the issue.

Reply:
You're the docs master so I trust your judgment :)

Reply:
Then I'll leave my name at the end, both as a matter of humility and to reflect reality. :)

Thanks, Marcin.